concepts/case/about.md

```diff
@@ -0,0 +1,129 @@
+# Case
+
+[Case][case] (often referred to as switch in other languages) is a form of control expression like if-else.
+Case allows for chaining of multiple if-else-if statements and can be more readable and also allows for powerful constructs.
+
+A case is defined by the keyword `case` followed by an optional expression.
+Then for each case, the keyword `when` is used followed by an expression which is compared to the case expression.
+The `when` keyword should not be indented from the `case` keyword.
+After the `when` keyword is the code that should be executed if the case expression matches the when expression.
+Case allows for an optional `else` statement which is executed if no other case matches.
+
+The case expression is evaluated and then compared to each `when` expression.
+The expression is compared using the case subsumption operator (`===`).
```

Comment:
In Ruby, we call that the "case equality operator".

Reply:
I have seen Jorg (from stack overflow) use it in 2010 and someone else call it this in their blog in 2022, but this is only the third time I have seen this `===` called that.